### ISSUE TITLE
Issue with Angular module

### ISSUE THREAD

**Issue by josevargasgalan** [fasttime/Polytype#4]
When i use Polytype with Angular 8 i get this error:

**Unexpected value 'AppComponent' declared by the module 'AppModule'**

AppComponent: 

```
import { Component, OnInit } from '@angular/core';
import { ClassA } from './classA';
import { ClassB } from './classB';
import { classes } from 'polytype';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.sass']
})
export class AppComponent extends classes(ClassA, ClassB) implements OnInit  {

  constructor() {
    super(['hola'], ['adios']);
  }

  ngOnInit(): void {
    this.printA();
    this.printB();
  }
}
```

AppModule

```
import { BrowserModule } from '@angular/platform-browser';
import { NgModule } from '@angular/core';

import { AppComponent } from './app.component';

@NgModule({
  declarations: [
    AppComponent
  ],
  imports: [
    BrowserModule
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```
If I delete _**extends classes(ClassA, ClassB)**_  everything works correctly

**Comment by fasttime** [comment]
Thanks for reporting this! I am able to reproduce the described behavior, it should be fixed in the next release.

**Comment by josevargasgalan** [comment]
You are welcome! I look forward to the change.

**Comment by fasttime** [comment]
This should be now fixed in version 0.5.0. Let me know if you are having any other problems.
